.github/settings.yml

```diff
@@ -0,0 +1,141 @@
+repository:
+  # See https://developer.github.com/v3/repos/#edit for all available settings.
+
+  # The name of the repository. Changing this will rename the repository
+  name: infra
+
+  # A short description of the repository that will show up on GitHub
+  description: nix-community infrastructure [maintainer=@Mic92]
+
+  # A URL with more information about the repository
+  homepage: "https://nix-community.org"
+
+  # A comma-separated list of topics to set on the repository
+  topics: [terraform, nixos]
```

Comment:
Is the app working, these haven't been added to the repo?

Reply:
Fixed in https://github.com/nix-community/infra/commit/433613dd0eb96849086bf34722fa1ed10ef0a1fb . Turns out it's stringly typed.